rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import nodeResolve from "rollup-plugin-node-resolve";
+import visualizer from "rollup-plugin-visualizer";
+import {uglify} from "rollup-plugin-uglify";
+
+const version = require("./package.json").version;
+const banner = [
+  "/*!",
+  ` * Application Insights JavaScript SDK - Core, ${version}`,
+  " * Copyright (c) Microsoft and contributors. All rights reserved.",
+  " */"
+].join("\n");
+
+const browserRollupConfigFactory = isProduction => {
+  const browserRollupConfig = {
+    input: "dist-esm/applicationinsights-core-js.js",
+    output: {
+      file: "browser/applicationinsights-core-js.js",
+      banner: banner,
+      format: "umd",
+      name: "aicore",
+      sourcemap: true
+    },
+    plugins: [
+      nodeResolve({
+        module: true,
+        browser: true,
+        preferBuiltins: false
+      })
+    ]
+  };
+
+  if (isProduction) {
+    browserRollupConfig.output.file = "browser/applicationinsights-core-js.min.js";
```

Comment:
@jpiyali Don't need to, but want to avoid confusion with node sdk. If I do remove the -js, should we also remove it from package name for consistency?

Reply:
as we dicussed, lets not remove in this check-in